github/ci/prow-deploy/files/jobs/kubevirt/vm-import-operator/OWNERS

```diff
@@ -5,8 +5,10 @@ filters:
       - machacekondra
       - masayag
       - pkliczewski
+      - mansam
```

Comment:
Please note that if you had an OWNERS file in kubevirt/vm-import-operator, this OWNERS file here would automatically be in sync with the OWNERS from kubevirt/vm-import-operator.

Reply:
Ah, I see, thanks. I'll remedy that in `kubevirt/vm-import-operator`.